Utils/componentHandler/index.d.ts

```diff
@@ -57,10 +57,10 @@ declare function help<TYPE extends 'command' | 'category' | 'all'>(
 
 declare function infoCMDs<
   ID extends Snowflake,
-  MODE extends 'kick' | 'ban' | 'delete',
+  MODE extends 'kick' | 'ban' | 'delete' | 'addToServer' | 'addToServerCMD',
   ENTITY_TYPE extends 'members' | 'emojis' | 'roles'
 >(
-  this: ButtonInteraction & { customId: `infoCMDs.${ID}.${MODE}.${ENTITY_TYPE}` },
+  this: ButtonInteraction | ModalSubmitInteraction & { customId: `infoCMDs.${ID}.${MODE}.${ENTITY_TYPE}` },
```

Comment:
note: does the `&` actually go for `ButtonInteraction` and `ModalSubmitInteraction` (as it is intended)?